src/TwilioChannel.php

```diff
@@ -56,6 +54,9 @@ public function send($notifiable, Notification $notification)
                 throw CouldNotSendNotification::invalidMessageObject($message);
             }
 
+            $to = $this->getTo($notifiable, $notification, $message);
```

Comment:
I commented on the last PR, but shouldn't we merge the additional tos instead of overridding?

Also, could you fix the tests please?

Reply:
1. We shouldn't. As any message should be sent to the user through single phone number.

2. I am working on the tests.

Reply:
What's the hold up on this PR?

Reply:
What happened with this PR? If this is not going anywhere, can I make a PR with proper tests for this patch?